<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="txtHost">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPort">
        <property name="text">
         <string>8080</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtUsername">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConnect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDisconnect">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtTopicSub">
        <property name="text">
         <string>Topic to subscribe</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnSubscribe">
        <property name="text">
         <string>Subscribe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnUnsubscribe">
        <property name="text">
         <string>Unsubscribe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxChat">
        <property name="title">
         <string>Message Area</string>
        </property>
        <widget class="QListWidget" name="listWidgetLog">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>361</width>
           <height>221</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtTopicPub">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>30</y>
           <width>113</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Topic to publish</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtMessage">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>70</y>
           <width>113</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>Enter message...</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnSend">
         <property name="geometry">
          <rect>
           <x>680</x>
           <y>30</y>
           <width>90</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnBrowseFile">
         <property name="geometry">
          <rect>
           <x>680</x>
           <y>70</y>
           <width>90</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>File...</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
